lib/proxy/log.rb

```diff
@@ -35,6 +30,7 @@ def self.logger
         logger = ::Logger.new(STDOUT)
       elsif log_file.casecmp('SYSLOG').zero?
         begin
+          @@needs_log_rolling = false
```

Comment:
This appears to be unused, should it be used instead of `@logger.instance_of?` in `handle_log_rolling`?